makedist

```diff
@@ -40,8 +40,8 @@ if test "${2}" -lt "13" -o "${2}" -eq "13" -a "${3}" -lt "5"; then
 fi
 IFS="$old_IFS"
 
-if test "x$PHPROOT" = "x"; then
-    PHPROOT=git@git.php.net:php-src.git;
+if test "x$PHPROOT" -ne "x"; then
+    PHPROOT=--remote=$PHPROOT
```

Comment:
Why adding `--remote` in the env ?  isn't a bit cleaner to only have the URI there ?

Reply:
That is a hack to allow local export as well as remote. Cleaner solution would be to introduce a new internal variable (e.g. `$remote_option` as in PHP-7.4+), but I'm really bad at bash scripting.

Reply:
I have just pushed a change to use an own variable for this.